src/cluster/Manager.cc

```diff
@@ -2,13 +2,45 @@
 
 #include "zeek/cluster/Manager.h"
 
+#include "zeek/Attr.h"
 #include "zeek/Func.h"
+#include "zeek/Scope.h"
 #include "zeek/cluster/Serializer.h"
 #include "zeek/cluster/websocket/WebSocket.h"
 #include "zeek/util.h"
 
+#include "Desc.h"
+
 using namespace zeek::cluster;
 
+void detail::report_non_functional_broker_tables(const zeek::EnumValPtr& cluster_backend_val) {
```

Comment:
this is probably a bit overcautious - but should this have an assert checking that the backend is != broker?